packages/reporter/src/main.tsx

```diff
@@ -120,10 +118,7 @@ class Reporter extends Component<SingleReporterProps> {
       statsStore,
     })
 
-    if (!runnerListenersAdded) {
```

Comment:
Now we are correctly cleaning up the reporter, we do not need this anymore.

`componentDidMount` will be called every time we visit the spec page and remount the reporter. Previously, we used the *same* event manager, instead of correctly tearing dow and creating a new one, so I'd added this check. Now we correctly clean up and recreate the event manager, we need to listen to it again (since it's a fresh instance).